client/src/main/java/io/pravega/client/stream/TimeWindow.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Could you remove the year from the copyright line to comply with the accepted header across all new files?

```
Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0
```

https://github.com/pravega/pravega/blob/master/HEADER